test/Base_Tests/src/Semantic/Arrow_Interop_Spec.enso

```diff
@@ -0,0 +1,41 @@
+from Standard.Base import all
+import Standard.Base.Errors.Common.Index_Out_Of_Bounds
+import Standard.Base.Errors.Common.Unsupported_Argument_Types
+import Standard.Base.Data.Polyglot_Array_Builder.Polyglot_Array_Builder
+
+from Standard.Test import all
+
+foreign arrow new_arrow = """
+    new[Int8]
+
+Polyglot_Array_Builder.set self index value = Polyglot_Array_Builder.set_builtin self index value
```

Comment:
I can remove this if people object. This is mostly for testing the functionality and it is not part of the official API